src/server_manager/web_app/app.ts

```diff
@@ -505,13 +505,16 @@ export class App {
     };
     const oauthUi = this.appRoot.getDigitalOceanOauthFlow(signOutAction);
     for (;;) {
-      const status = await this.digitalOceanRetry(async () => {
+      const info = await this.digitalOceanRetry(async () => {
         if (cancelled) {
           throw CANCELLED_ERROR;
         }
         return await digitalOceanAccount.getStatus();
       });
-      if (status === digitalocean.Status.ACTIVE) {
+      if (info.warning) {
+        this.appRoot.showError(`DigitalOcean warning: ${info.warning}`);
```

Comment:
Needs i18n.

Reply:
Also, should we show this with the error in this.appRoot.showError instead?
I find it odd to have a warning pop up like that. We only use it for errors.

Reply:
I'm not sure what you're suggesting here.

Going through the flow myself, I do think the popup notification seemed natural. 
 It does not block the UI, but it will not disappear until it is explicitly dismissed.